pkg/storegateway/bucket_e2e_test.go

```diff
@@ -664,7 +664,7 @@ func TestBucketStore_Series_ChunksLimiter_e2e(t *testing.T) {
 					} else {
 						assert.Error(t, err)
 						assert.Contains(t, err.Error(), testData.expectedErr)
-						status, ok := status.FromError(err)
+						status, ok := grpcutil.ErrorToStatus(err)
```

Comment:
[nit]
Another possibility would be:

```suggestion
	code := grpcutil.ErrorToStatusCode(err)
```

and then to check 
```
require.Equal(t, codes.Unavailable, code)
```